From 29af598733bd962c9ec4d2cfbd5cb5f4122a9162 Mon Sep 17 00:00:00 2001
From: Jeegar Patel <jeegar.patel@xilinx.com>
Date: Tue, 8 Sep 2020 01:37:28 -0700
Subject: [PATCH 1570/1851] v4l: xilinx: vip: Fix UNINIT warnings

commit f0c5dfad64e3f67db76e76933ae30ea21a9ec566 from
https://github.com/Xilinx/linux-xlnx.git

This patch fix UNINIT type coverity warnings.

Warnings:
Event uninit_use: Using uninitialized value "vf_code".
Event uninit_use: Using uninitialized value "width".

Signed-off-by: Jeegar Patel <jeegar.patel@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-vip.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/platform/xilinx/xilinx-vip.c b/drivers/media/platform/xilinx/xilinx-vip.c
index 4da7acb130c9..4c8603064a74 100644
--- a/drivers/media/platform/xilinx/xilinx-vip.c
+++ b/drivers/media/platform/xilinx/xilinx-vip.c
@@ -263,9 +263,9 @@ EXPORT_SYMBOL_GPL(xvip_width_padding_factor);
 const struct xvip_video_format *xvip_of_get_format(struct device_node *node)
 {
 	const char *pattern = "mono";
-	unsigned int vf_code;
+	unsigned int vf_code = 0;
 	unsigned int i;
-	u32 width;
+	u32 width = 0;
 	int ret;
 
 	ret = of_property_read_u32(node, "xlnx,video-format", &vf_code);
-- 
2.31.1

